### PR TITLE
Shield Telemetry

### PR DESCRIPTION
**Pull request by ianb**
This PR also includes a null add-on for the study

### CODE REVIEW

File: addon/shield-setup.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+          send: true,
+          // Marks pings with testing=true.  Set flag to `true` before final release
+          removeTestingFlag: false,
+        },
+        endings: {
+          /** standard endings */
+          "user-disable": {
+            baseUrls: [
+              // FIXME: put in correct URL:
+              "https://qsurvey.mozilla.com/s3/Shield-Study-Example-Survey/?reason=user-disable",
```
Comment (johngruen) [inline addon/shield-setup.js]: @ianb https://qsurvey.mozilla.com/s3/side-view-shield-study

File: addon/manifest.json.tmpl [hunk 2]
```diff
@@ -22,19 +22,19 @@
...
       {{#SHIELD}}
       "study": {
-        "schema": "../experiment/study/schema.json",
+        "schema": "./experiment/study/schema.json",
         "parent": {
           "scopes": ["addon_parent"],
-          "script": "../experiment/study/api.js",
-          "paths": [["study", "studyDebug"]]
+          "script": "./experiment/study/api.js",
+          "paths": [["study"], ["studyDebug"]]
```
Comment (fzzzy) [inline addon/manifest.json.tmpl]: Was this the typo you mentioned? Must have been tough to find.

File: package.json [hunk 3]
```diff
@@ -37,16 +37,18 @@
     "lint:styles": "stylelint ./addon/*.css",
     "build": "npm-run-all build:*",
     "build:ga": "mkdir -p addon/build && cp $(node -e 'console.log(require.resolve(\"testpilot-ga\"))') addon/build/testpilot-ga.js",
-    "build:testutils": "if [[ -n \"$SHIELD\" ]] ; then copyStudyUtils ./experiment/ ; else rm -rf ./experiment/study/ ; fi",
+    "build:testutils": "if [[ -n \"$SHIELD\" ]] ; then copyStudyUtils ./addon/experiment/ ; else rm -rf ./addon/experiment/study/ ; fi",
```
Comment (fzzzy) [inline package.json]: where is copyStudyUtils defined? i only see it being called.

Comment (fzzzy) [reply package.json]: Ok, I see it is in an npm dependency shield-study-addon-utils.

**Pull request merged**